#!/usr/bin/env python3

PACKAGE = "livox_transform"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("tran_x", double_t, 0, "Translation X", 0.0, -10.0, 10.0)
gen.add("tran_y", double_t, 0, "Translation Y", 0.0, -10.0, 10.0)
gen.add("tran_z", double_t, 0, "Translation Z", 0.0, -10.0, 10.0)
gen.add("theta_x", double_t, 0, "Rotation X", 0.0, -3.1415926, 3.1415926)
gen.add("theta_y", double_t, 0, "Rotation Y", 0.0, -3.1415926, 3.1415926)
gen.add("theta_z", double_t, 0, "Rotation Z", 0.0, -3.1415926, 3.1415926)

exit(gen.generate(PACKAGE, PACKAGE, "Test"))